recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux-libc-headers/linux-libc-headers_%.bbappend

```diff
@@ -1,5 +1,9 @@
 RENESAS_BSP_URL = "git://github.com/xen-troops/linux.git"
```

Comment:
WHY is this patch if it is marked as [do not merge]????

Reply:
I will remove it from PR